### ISSUE TITLE
In-line cursor not visible

### ISSUE THREAD

**Issue by AppleFromHell** [notion-enhancer/notion-enhancer#484]
FILL OUT THIS TEMPLATE, REMOVING/REPLACING ALL NON-BOLD TEXT WITH YOUR OWN INFORMATION.
BE AS CLEAR AND CONCISE AS POSSIBLE.

**describe the bug**
Whenever I am typing something, there is no in-line cursor that says where you are in the document you are typing. However, this does not appear to happen all the time. I have no clue why it sometimes does and sometimes does not happen.

**expected behaviour**
I expect to see where my nexy character will be located, and can thus also navigate with the arrow keys if needed.

**screenshots**
Not applicable

**platform**
Windows 10

**Comment by dragonwocky** [comment]
this is caused by tabs - switching focus out of and then back to notion or disabling tabs are temporary solutions until it's fixed in an update.

**Comment by arjun-khatri** [comment]
Yes, I am facing the same problem. At the startup or when I change the theme of Notion, the cursor is not visible. I have to minimize the Notion by clicking on the enhancer icon in the taskbar and when opened again the cursor appears. Please help me with it, I have to do this every time I turn on the PC.
